test/src/receiver/main.cpp

```diff
@@ -28,102 +28,148 @@ using TIME = NDTime;
+    };
+    cadmium::dynamic::modeling::Models submodels_TOP = {
+            generator, receiver1
+    };
+    cadmium::dynamic::modeling::EICs eics_TOP = {
 
-std::shared_ptr<cadmium::dynamic::modeling::model> receiver1 = cadmium::dynamic::translate::make_dynamic_atomic_model<Receiver, TIME>("receiver1");
+    };
+    cadmium::dynamic::modeling::EOCs eocs_TOP = {
+        cadmium::dynamic::translate::make_EOC
+            <receiver_defs::out,outp>("receiver1")
+    };
+    cadmium::dynamic::modeling::ICs ics_TOP = {
+        cadmium::dynamic::translate::make_IC
+            <iestream_input_defs<Message_t>::out,receiver_defs::in>
+                ("generator","receiver1")
+    };
+    std::shared_ptr<cadmium::dynamic::modeling::coupled<TIME>> TOP =
+        std::make_shared<cadmium::dynamic::modeling::coupled<TIME>>(
+            "TOP",submodels_TOP,iports_TOP,
+                oports_TOP,eics_TOP,eocs_TOP,ics_TOP);
 
+    ///****************////
 
-/************************/
-/*******TOP MODEL********/
-/************************/
-cadmium::dynamic::modeling::Ports iports_TOP = {};
-cadmium::dynamic::modeling::Ports oports_TOP = {typeid(outp)};
-cadmium::dynamic::modeling::Models submodels_TOP = {generator, receiver1};
-cadmium::dynamic::modeling::EICs eics_TOP = {};
-cadmium::dynamic::modeling::EOCs eocs_TOP = {
-  cadmium::dynamic::translate::make_EOC<receiver_defs::out,outp>("receiver1")
-};
-cadmium::dynamic::modeling::ICs ics_TOP = {
-  cadmium::dynamic::translate::make_IC<iestream_input_defs<Message_t>::out,receiver_defs::in>("generator","receiver1")
-};
-std::shared_ptr<cadmium::dynamic::modeling::coupled<TIME>> TOP = std::make_shared<cadmium::dynamic::modeling::coupled<TIME>>(
- "TOP", 
- submodels_TOP, 
- iports_TOP, 
- oports_TOP, 
- eics_TOP, 
- eocs_TOP, 
- ics_TOP 
-  );
-
-///****************////
-
-    auto elapsed1 = std::chrono::duration_cast<std::chrono::duration<double, std::ratio<1>>>(hclock::now() - start).count();
+    auto elapsed1 = std::chrono::duration_cast<std::chrono::duration
+        <double,std::ratio<1>>>(hclock::now() - start).count();
     cout << "Model Created. Elapsed time: " << elapsed1 << "sec" << endl;
     
     cadmium::dynamic::engine::runner<NDTime, logger_top> r(TOP, {0});
-    elapsed1 = std::chrono::duration_cast<std::chrono::duration<double, std::ratio<1>>>(hclock::now() - start).count();
+    elapsed1 = std::chrono::duration_cast<std::chrono::duration
+        <double,std::ratio<1>>>(hclock::now() - start).count();
     cout << "Runner Created. Elapsed time: " << elapsed1 << "sec" << endl;
 
     cout << "Simulation starts" << endl;
 
     r.run_until(NDTime("04:00:00:000"));
-    auto elapsed = std::chrono::duration_cast<std::chrono::duration<double, std::ratio<1>>>(hclock::now() - start).count();
+    auto elapsed = std::chrono::duration_cast<std::chrono::duration
+        <double, std::ratio<1>>>(hclock::now() - start).count();
     cout << "Simulation took:" << elapsed << "sec" << endl;
```

Comment:
Spaces before and after << operator can be removed.

Reply:
Solved

Reply:
There is a space between << and endl. It can be removed. (line 151)

Reply:
solved